### DISCUSSION TITLE
Does NX affected uses git 2 dots diff or 3 dots diff to determine changed files

### DISCUSSION THREAD

**Discussion by vinayak-ag** [nrwl/nx#26478] [Questions]
nx affected:test --base=origin/<dev-branch> --head=HEAD

On using this NX command for a pull request.. Does it will take the git diff from merge base of feature branch and base branch.. That means just the files changed by the PR and not the diff between HEAD of base branch and HEAD of PR?
Thanks in advance!

**Comment by jaysoo** [comment]
We leave out the dots, so essentially the two-dot form. In practice, I don't think it matters since if you are using `git diff A B` or `git diff A...B`, and `A` is an ancestor of `B`, then both forms will yield the same result.
